### PR TITLE
audio embeddings update

### PR DESCRIPTION
**Pull request by krishna1priya**

### CODE REVIEW

File: lyrics_audio_embeddings.py [hunk 1]
```diff
@@ -6,100 +6,116 @@
...
-            # Generate embedding for each chunk and average them
             chunk_embeddings = []
             with torch.no_grad():
                 for chunk in input_ids_chunks:
                     chunk_embedding = text_transformer(chunk)
                     chunk_embeddings.append(chunk_embedding)
 
-            # Average embeddings across chunks
             aggregated_embedding = torch.mean(torch.stack(chunk_embeddings), dim=0)
-            embeddings.append(aggregated_embedding.detach().numpy())
```
Comment (Ranvirsv) [inline lyrics_audio_embeddings.py:58]: we need detach to make a copy
If we are using it as a separate variable. 

Else it will go into the gradient calculations.

File: lyrics_audio_embeddings.py [hunk 2]
```diff
@@ -6,100 +6,116 @@
...
+            waveform = resampler(waveform)
+
+        # Normalize waveform
+        waveform = waveform / (waveform.abs().max() + 1e-8)
+        waveform = waveform.to(device)
+
+        # Generate embedding
+        with torch.no_grad():
+            embedding = audio_transformer(waveform)
+            return embedding.cpu().numpy()
```
Comment (Ranvirsv) [inline lyrics_audio_embeddings.py:54]: Detach ? 
Please check if we need this

**Review by Ranvirsv — COMMENTED** [review]
Please address the comments
